src/lib.rs

```diff
@@ -564,17 +564,11 @@ pub enum AuthType {
 /// ```ignore
 /// let revocation_response = client
 ///     .revoke_token(AccessToken::new("some token".to_string()).into())
-///     .request(...)
-///     .unwrap();
-///
-/// assert!(matches!(revocation_response, Err(
-///     RequestTokenError::ServerResponse(
-///         BasicRevocationErrorResponse{
-///             error: RevocationErrorResponseType::UnsupportedTokenType,
-///             ..
-///         })
-///     )
-/// ));
+///     .request(...);
```

Comment:
since `revoke_token` returns a `Result` now, the example would need `unwrap`/`expect` before calling `request`. might be worth replacing `ignore` with `rust` and trying to get the example to compile in order to prevent future inconsistencies

Reply:
Will do.

Reply:
Done.